src/app/technical-record/brakes/brakes.component.html

```diff
@@ -19,3 +19,7 @@
       <vtm-axle-brakes [axle]="axle" [vehicleType]="currentVehicleType"></vtm-axle-brakes>
   </ng-container>
 </ng-container>
+
+<ng-container *ngIf="currentVehicleType === 'psv'">
```

Comment:
Do we have test to cover when the vehicle type is not PSV from the perspective of the parent i.e. `brakes`